pkg/output/result.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package output
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	//output types
+	JSONAlpha1 = "json-alpha1"
+	Text       = "text"
+)
+
+// Result represents the final result
+type Result struct {
+	Passed bool  `json:"passed"`
+	Logs   []Log `json:"logs"`
+}
+
+// Log represents the logs which are used to return the final result in the JSON format
+type Log struct {
```

Comment:
This could be named better since these are not logs.

Reply:
hun, for me it shows a list of logs with levels(info/warn/error). However, what name we would like to suggest?

Reply:
Changed to Output.